modules/machineset-troubleshooting-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * storage/persistent_storage/persistent-storage-azure.adoc
+// * storage/persistent_storage/persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "creating-machineset-azure"]
+:mapi:
+endif::[]
+ifeval::["{context}" == "persistent-storage-azure"]
+:pvc:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:pvc:
+endif::[]
+
+:_content-type: REFERENCE
+[id="machineset-troubleshooting-azure-ultra-disk_{context}"]
+= Troubleshooting resources for machine sets that enable ultra disks
+
+Use the information in this section to understand and recover from issues you might encounter.
+
+ifdef::pvc[]
+[id="ts-pvc-mounting-ultra_{context}"]
+== Unable to mount a persistent volume claim backed by an ultra disk
+
+If there is an issue mounting a persistent volume claim backed by an ultra disk, the pod becomes stuck in the `ContainerCreating` state and an alert is triggered.
+
+For example, if the `UltraSSDEnabled` parameter is not set on the machine that backs the node that hosts the pod, the following error message appears:
```

Comment:
Little nitpick 👼 . Let's change `UltraSSDEnabled` to `additionalCapabilities.ultraSSDEnabled`, otherwise it might be trick users into thinking it's a setting on the MachineSet (that one is the `ultraSSDCapability`), whereas this is the additional capability that ends up being set by the Azure SDK on the Azure VM backing the MAPI Machine.